.gitignore

```diff
@@ -16,3 +16,5 @@ profile.coverprofile
 explain_test
 cmd/explaintest/explain-test.out
 _tools/
+*.fail.go
+binary.go.tmp
```

Comment:
"binary.go.tmp" is generated by gofail?

Reply:
On my mac, I can not reproduce how to generate this file, so I remove this item from .gitignore.